app/assets/javascripts/angular/controllers/topics_ctrl.coffee

```diff
@@ -15,6 +15,15 @@ angular.module "lean-coffee"
       if confirm('Are you sure you want to delete this?')
         topicsService.delete(topic)
 
+    @archive = (topic) =>
+      if confirm('Are you sure you want to archive this?')
+        topicsService.archive(topic)
+
+    @archiveAll = =>
+      if confirm('Are you sure you want to archive all discussed topics?')
+        topicsService.archiveAll()
+        topicsService.reload()
```

Comment:
No worries! I tracked it down and fixed it all up in 8c1a461. Which also allowed removing the `reload` call here. I think both I and the code are better now because of it! :smile: